main.go

```diff
@@ -32,8 +35,14 @@ func init() {
 		fmt.Println(app.Name)
 		fmt.Println(c.String("command"))
 
+		ex, err := os.Executable()
+		if err != nil {
+			panic(err)
```

Comment:
This function (`fromFile`) returns an error.  Every other error-checking branch here returns an error.  This should return an error as well, so that the caller (the `saved-command` sub-command) can handle the error as it sees fit.

Reply:
updated